Berksfile

```diff
@@ -3,11 +3,6 @@ source 'https://supermarket.chef.io'
 metadata
 
 group :integration do
-  cookbook 'apt'
   cookbook 'fqdn', git: 'https://github.com/drpebcak/fqdn-cookbook.git'
```

Comment:
we include fqdn in the runlist in testing. It's probably something we could wrap into the setup recipe there though